lld/ELF/ScriptParser.cpp

```diff
@@ -1411,6 +1411,7 @@ static std::optional<uint64_t> parseFlag(StringRef tok) {
       .Case(CASE_ENT(SHF_COMPRESSED))
       .Case(CASE_ENT(SHF_EXCLUDE))
       .Case(CASE_ENT(SHF_ARM_PURECODE))
+      .Case(CASE_ENT(SHF_AARCH64_PURECODE))
```

Comment:
Update the test test/ELF/input-section-flags.s to test SHF_AARCH64_PURECODE

(rebase. I am adding a SHF_ARM_PURECODE test there.)